testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oauth/ServiceAccountTest.java

```diff
@@ -281,6 +291,25 @@ public void clientCredentialsAuthRequest_ClientES256_RealmPS256() throws Excepti
     	conductClientCredentialsAuthRequest(Algorithm.HS256, Algorithm.ES256, Algorithm.PS256);
     }
 
+    @Test
+    public void failManagePassword() {
+        UserResource serviceAccount = adminClient.realm("test").users().get(userId);
+        UserRepresentation representation = serviceAccount.toRepresentation();
+
+        CredentialRepresentation password = new CredentialRepresentation();
+        password.setValue("password");
+        password.setType(CredentialRepresentation.PASSWORD);
+        password.setTemporary(false);
+
+        representation.setCredentials(Arrays.asList(password));
+
+        this.expectedException.expect(Matchers.allOf(Matchers.instanceOf(ClientErrorException.class), 
+                Matchers.hasProperty("response", Matchers.hasProperty("status", Matchers.is(409)))));
```

Comment:
It may look weird a `409` response here. But I didn't want to change the scope of this PR and review how the endpoint is mapping errors to the proper status.